### DISCUSSION TITLE
Feature request: allow safe use of secrets inside webhooks

### DISCUSSION THREAD

**Discussion by roelof-adam** [community/community#77089] [Apps, API and Webhooks]
### Select Topic Area

Product Feedback

### Body

Github webhooks is a great feature, however, we need to send requests to a number of endpoints that require an authorization header. It would be great if we could refer to secrets that are stored in github environments or github secrets, something like:

`https://some-endpoint-url/generic-webhook-trigger/invoke?token=${{SECRET}}`

**Comment by ghostinhershell** [comment]
Hey there @roelof-adam, 

Thanks so much for sharing your feature request! I'll file this with the relevant teams here at GitHub so it can get more visibility ✅

**Reply by roelof-adam** [reply]
Thanks, that'd be great
